cmd/homi/setup/flags.go

```diff
@@ -53,14 +53,14 @@ var (
 	}
 
 	numOfCNsFlag = cli.IntFlag{
-		Name:  "num",
+		Name:  "cn-num",
```

Comment:
```suggestion
		Name:  "num-cn",
```
to pair with "num-validators"

Reply:
This is paired with "en-num", "pn-num".
So I will pair num-validators. :)

Reply:
Done.